app/experimenter/experiments/tests/test_views.py

```diff
@@ -256,6 +256,23 @@ def test_list_filters_by_search_text(self):
             set(second_response_context["experiments"]), set([exp_3])
         )
 
+    def test_filters_by_review_in_qa(self):
+        exp_1 = ExperimentFactory.create_with_variants(
+            review_qa_requested=True, review_qa=False
+        )
+        ExperimentFactory.create_with_variants(
+            review_qa_requested=False, review_qa=False
+        )
+        ExperimentFactory.create_with_variants(
+            review_qa_requested=True, review_qa=None
```

Comment:
When does this case happen?

Reply:
Yea this was kind of nonsensical so I got rid of the `None`.